pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -674,6 +674,8 @@ type ComponentSettings struct {
 	// the `LoadBalancer` expose strategy is used. This is not effective if a different expose
 	// strategy is configured.
 	NodePortProxyEnvoy NodeportProxyComponent `json:"nodePortProxyEnvoy"`
+	// KonnectivityProxy configures resources limits/requests for konnectivity-server sidecar
```

Comment:
super super uber nitpick, but there's a period missing at the end of the sentence.